index.php

```diff
@@ -153,6 +153,11 @@
 	define('PHPBB_32', true);
 }
 
+if (file_exists($phpbb_root_path . 'phpBB/cron/manager.' . $phpEx))
+{
+	define('PHPBB_33', true);
```

Comment:
Other than in the above change, will the logic here to determine the version be used elsewhere?

Reply:
yes, the following commit.

Reply:
If we're removing the usage of the `PHPBB_32` and `PHPBB_33` constants from the code then these constant assignments should just be removed.